docs/libcurl/curl_easy_setopt.3

```diff
@@ -367,6 +367,9 @@ This HTTP/2 stream depends on another exclusively. See
 \fICURLOPT_STREAM_DEPENDS_E(3)\fP
 .IP CURLOPT_STREAM_WEIGHT
 Set this HTTP/2 stream's weight. See \fICURLOPT_STREAM_WEIGHT(3)\fP
+.IP CURLOPT_FORM_ESCAPE_AS_MIME
+Use backslash-escaping in form field and file names rather than
+percent-escaping. See \fICURLOPT_FORM_ESCAPE_AS_MIME(3)\fP
```

Comment:
I think maybe the name could be improved but I don't have a better suggestion just now.

I also think it is unfortunate to do this as an option to `curl_easy_setopt()` and not in the `curl_mime_*()` API, but since we don't have a `curl_mime_setopt()` there's no other good place. The question is perhaps if a `curl_mime_setopt()` function should be added?

Another alternative is to make it a slightly more generic formpost option like `CURLOPT_FORM_OPTION` which takes a bitmask and this would be the first defined bit in that bitmask: `CURLFOPT_BACKSLACHESCAPE`.

Reply:
> I think maybe the name could be improved but I don't have a better suggestion just now.

Agreed. I named it `CURL_FORM_ESCAPE_LEGACY` first, but this did not satisfied me.

> Another alternative is to make it a slightly more generic formpost option like CURLOPT_FORM_OPTION which takes a bitmask.

Good idea. But I hope we would not have too many of them in the future!

Reply:
Done, as `CURL_MIME_OPTIONS` with bitmask `CURLMIMEOPT_FORMESCAPE`.